.github/workflows/create_rc_pr.yml

```diff
@@ -24,7 +24,19 @@ jobs:
                 pip install -r requirements.txt
                 pip install -r tasks/libs/requirements-github.txt
 
-            - name: Create RC PR
+            - name: Determine the release active branch
```

Comment:
Is this change expected to be part of the PR? I don't mind it but it seems the PR is only about the task itself